crates/wasmtime/src/runtime/component/values.rs

```diff
@@ -1000,3 +1031,12 @@ fn unexpected<T>(ty: InterfaceType, val: &Val) -> Result<T> {
         val.desc()
     )
 }
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FutureAny(pub(crate) u32);
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct StreamAny(pub(crate) u32);
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ErrorContextAny(pub(crate) u32);
```

Comment:
How temporary is the representation of these structures? E.g. is the `u32` what it'll be in the long run?

Reply:
They're like resource reps in that they'll be `u32`s for wasm32 but become `u64`s when we add wasm64 support for the component model.